### DISCUSSION TITLE
Manual request

### DISCUSSION THREAD

**Discussion by ruslan61rus** [Ralim/IronOS#1221] [General]
Can anybody help me to find manual, how i can to make settings in device, e.t.c Is there manual? On TS100 found, but i think, menu can have some difference. Today i got pinecil from Aliexpress, i do not now, how it use. Thank you!

**Comment by paulfertser** [comment]
Hi,

On Tue, Feb 08, 2022 at 08:42:01AM -0800, ruslan61rus wrote:
> Can anybody help me to find manual, how i can to make settings in device, e.t.c
> Is there manual? On TS100 found, but i think, menu can have some difference.
> Today i got pinecil from Aliexpress, i do not now, how it use. Thank you!

Are you sure it was Pinecil and not a clone? Settings are described
here:
https://github.com/Ralim/IronOS/blob/master/Documentation/Settings.md

I thought Pine doesn't sell anything via Aliexpress officially.

**Reply by ruslan61rus** [reply]
I bought here https://aliexpress.ru/item/1005003718319079.html?af=739_1010658&utm_campaign=739_1010658&aff_platform=api-new-link-generate&utm_medium=cpa&cn=12r6zws9t20mm4izw6b0m7766hv9nf4o&dp=12r6zws9t20mm4izw6b0m7766hv9nf4o&aff_fcid=dd15681431c94cdea49cfeef45afb010-1644339897314-05090-_AavLA9&cv=3&aff_fsk=_AavLA9&sk=_AavLA9&aff_trace_key=dd15681431c94cdea49cfeef45afb010-1644339897314-05090-_AavLA9&terminal_id=474bc3d2c28a474c9dd5ec8da37e3f2c&utm_source=aerkol&utm_content=3

**Reply by discip** [reply]
This is definitely not a genuine Pinecil, since it is already stated on the page, that the chip is a GD32!

**Reply by ruslan61rus** [reply]
There is GigaDevice GD32VF103TB\ 32-bit RV32IMAC RISC-V “Bumblebee Core” @ 108 MHz, please, read specifications here https://wiki.pine64.org/wiki/Pinecil#Pinecil_board_information.2C_schematics_and_certifications

**Reply by Ralim** [reply]
Hia,

So Pinecil's sold via AliExpress are not official and may be clones.

All documentation is in the docs folder of this GitHub.


https://github.com/Ralim/IronOS/tree/master/Documentation


For settings you probably want :
 https://github.com/Ralim/IronOS/blob/master/Documentation/Settings.md

**Reply by River-Mochi** [reply]
> There is GigaDevice GD32VF103TB\ 32-bit RV32IMAC RISC-V “Bumblebee Core” @ 108 MHz, please, read specifications here https://wiki.pine64.org/wiki/Pinecil#Pinecil_board_information.2C_schematics_and_certifications

you are quoting the specs from the official Pine64 website and not from AliExpress.

**Reply by ruslan61rus** [reply]
> > There is GigaDevice GD32VF103TB\ 32-bit RV32IMAC RISC-V “Bumblebee Core” @ 108 MHz, please, read specifications here https://wiki.pine64.org/wiki/Pinecil#Pinecil_board_information.2C_schematics_and_certifications
> 
> you are quoting the specs from the official Pine64 website and not from AliExpress.

From Ali all tha same OS: Ralim's IronOS build

Chipset: GigaDevice GD32VF103TB

CPU: 32-bit RV32IMAC RISC-V "Bumblebee Core" @ 108 MHz

**Comment by ruslan61rus** [comment]
At first look all works fine. Is there method, how i can identificate, original iron i bouht or clone one?

**Reply by Ralim** [reply]
If you bought on AliExpress we have no way to know.
We suspect they are probably 1:1 copies, but there is no good way to check every part was produced to the same specification

**Reply by River-Mochi** [reply]
Could you also tell me which version of Firmware it came with?  was it  2.14 or 2.15?   it would be helpful

**Reply by ruslan61rus** [reply]
> Could you also tell me which version of Firmware it came with? was it 2.14 or 2.15? it would be helpful

I am not sure, because on first start i flushed iron with 2.16. Flashing process was without any problem. Now, i already flashed it with 2.17. No issues occured.

**Reply by ruslan61rus** [reply]
Now, i already flashed it with 2.17. No issues occured. About box. What you think about that https://www.youtube.com/watch?v=R_rCIk0iIv0? There is box with clear plastic window))

**Reply by River-Mochi** [reply]
I only asked because I noticed on the clones that they come with very old 2.14 firmware or even older

**Reply by River-Mochi** [reply]
If it works and you already have it, you could just keep using it, if you run into issues, go to the Pine64- pinecil channel and sometimes the community volunteers can help you diagnose and fix problem.

**Comment by ruslan61rus** [comment]
![IMG_20220209_113425](https://user-images.githubusercontent.com/19176374/153159422-c037a1db-952c-41bb-a27d-8613e6238d0b.jpg)
![IMG_20220209_113357](https://user-images.githubusercontent.com/19176374/153159431-54d30fe1-7fda-4f31-aab4-520142fcf3e5.jpg)
There is pine on board, is it original? As i understand from this theme, is all the same https://ksger.com.ua/pinecilremont.html

**Reply by River-Mochi** [reply]
many of the ones on AliExpress are clones.  Pine64 made an announcement in the February Blog.   
These companies in china have completely copied the  Pinecil including the pine cone logos on the PCB and the outside of the plastic case. There are some easy ways to tell they are fake clones.  

- Here are a couple easy ways to tell it is fake:
(1)  if it did not come in a black box with a clear plastic window (FAKE)
(2) If it came in any box that was colored like Red or Blue box  (FAKE)
(3)  Pine64 has never made a Blue Pinecil - all blue are fake.
(4)  They also do not make a full Red Pinecil - they sell full Black ones only with light blue finger grip. they used to sell an additional plastic case/shell that was red and you could buy as an Add-on, but not as a new full pinecil.

**Reply by ruslan61rus** [reply]
![IMG_20220312_163448](https://user-images.githubusercontent.com/19176374/158021114-b2682e61-e61c-48be-866e-a66b460f5030.jpg)
47-th week 2021, is 22-28 November. I ordered in January 2022, so look like truth. Box is red.

**Reply by River-Mochi** [reply]
Thanks this is helpful.  Clones do seem to work because Pine64 published the  Schematic and the firmware is opensource. It is very easy for another company to make a copy.  But they are not supposed to copy the Trademark Pine64 and Pine cone (it is like buying a fake Gucci or Vuitton or Levi's).  They could make a no-name model but they want to use fake  Pine64 symbols to get same price as real Pine64.

The specs will be dubious because it was not made by Pine64. For sure it will not be supported by official Pine64, but if you are going to keep it and run into issues,  you could ask the  community volunteers in the Pine64 forum - Pinecil channel  which is in discord but has a bridge to telegram/matrix too. 
the Pinecil channel is very active.  make sure to update to newest  IronOS firmware.  One issue people run into is the Clones have a very old firmware and then they can't update to 2.17.   but if this happens to you,  join the Pinecil chat channel and any volunteer around will be able to help - it's usually rework soldering the OLED screen to fix that.

**Comment by River-Mochi** [comment]


**Reply by ruslan61rus** [reply]
Ok. thank you!

**Reply by River-Mochi** [reply]
would you mind providing a photo of the back of the PCB?  I'm trying to confirm a theory we have about the clone fake ones.  what is the date stamp on the PCB  it will be a number like  2054  and is near the U14  spot on the underside of the PCB.

**Reply by River-Mochi** [reply]
I wrote a Guide to avoid fake Pinecil  [1257](https://github.com/Ralim/IronOS/discussions/1257#discussion-3934040l)
